src/cp/cp.rs

```diff
@@ -988,13 +984,6 @@ fn copy_file(source: &Path, dest: &Path, options: &Options) -> CopyResult<()> {
         println!("{}", context_for(source, dest));
     }
 
-    let mut preserve_context = false;
```

Comment:
Do you mind commenting this out or something and mention that it is for the `context` argument (which hasn't been implemented yet)?

Reply:
Done. Had to wrap it in a block for the allow macro to cover the reassignment.